### PR TITLE
Fix names inputs

### PR DESCRIPTION
**Pull request by shainw**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Insights/InputEntity_Account/WindowsSigninActivity.yaml [hunk 1]
```diff
@@ -210,5 +210,5 @@ Insights:
   Type: LineChart
 
  AdditionalQuery:
-  Text: "See All Windows Logons"
+  Text: "See all windows sign-ins"
```
Comment (ms-bnick) [inline Insights/InputEntity_Account/WindowsSigninActivity.yaml:213]: Windows should be capitalized as it is a proper noun

File: Insights/InputEntity_Host/EventLogClear.yaml [hunk 2]
```diff
@@ -78,7 +78,7 @@ BaseQuery: |
   HostClearedEventLog('{{Host_HostName}}', '{{Host_NTDomain}}', '{{Host_DnsDomain}}', '{{Host_AzureID}}', '{{Host_OMSAgentID}}')
 Insights:
  Id: 9a70a72d-25d4-7212-b73e-4f302a90c06a
- DisplayName: Event Logs Cleared on Host
+ DisplayName: Event Logs cleared on host
```
Comment (ms-bnick) [inline Insights/InputEntity_Host/EventLogClear.yaml:81]: Is "Logs" supposed to be capitalized?

File: Insights/InputEntity_Host/EventLogClear.yaml [hunk 3]
```diff
@@ -150,15 +150,15 @@ Insights:
       Query: "{{BaseQuery}} | {{RowFilter}}"
 
  ChartQuery: 
-  Title: "Log Clear over time"
+  Title: "Log clear over time"
```
Comment (ms-bnick) [inline Insights/InputEntity_Host/EventLogClear.yaml:153]: Log clear activity over time?

File: Insights/InputEntity_Host/GroupAdditions.yaml [hunk 4]
```diff
@@ -113,21 +113,21 @@ Insights:
...
-  Text: "See All Group Additions"
+  Text: "See all group additions"
   Query: "project TimeGenerated, EventID, Activity, Computer, MemberAdded, MemberName, MemberSid, TargetUserName, TargetDomainName, TargetSid, UserPrincipalName, SubjectUserName, SubjectUserSid, WellKnownGroupSID, WellKnownLocalSID, _ResourceId, SourceComputerId, timestamp, HostCustomEntity | order by TimeGenerated desc"
 
 Activities:
  EnabledByDefault: true
  Items:
    - Id: b880ad94-f905-4ba8-8a3f-9088b19b12fa
-     Description: Account added to Local Administrators Group
+     Description: Account added to Local Administrators group
```
Comment (ms-bnick) [inline Insights/InputEntity_Host/GroupAdditions.yaml:123]: Checked online.  In the AD Docs, local administrators group is not capitalized
https://docs.microsoft.com/en-us/azure/active-directory/devices/assign-local-admin
"To manage a Windows device, you need to be a member of the **local administrators group**. "

File: Insights/InputEntity_Host/ProcessEntropy.yaml [hunk 5]
```diff
@@ -187,16 +187,13 @@ Insights:
     Project:   "project Title = 'Execution by User Accounts', Process = case(array_length(Process) > 1, 'Many', array_length(Process) == 1, tostring(Process[0]), 'None'), EntropyWeight = case(array_length(Weight) > 1, strcat(tostring(round(avgWeight, 3)), ' avg'), array_length(Weight) == 1, tostring(Weight[0]), 'None'), ProcessCountOnHost = iff(isempty(Process), 0, ProcessCountOnHost)"
 
  ChartQuery: 
-  Title: "Rare Processes over Time (Weight <= 75)"
+  Title: "Rare processes over time (Weight <= 75)"
```
Comment (ms-bnick) [inline Insights/InputEntity_Host/ProcessEntropy.yaml:190]: Weight -> weight

File: Insights/InputEntity_Host/WindowsSigninActivity.yaml [hunk 6]
```diff
@@ -249,7 +249,7 @@ Insights:
   Type: LineChart
 
  AdditionalQuery:
-  Text: "See All Windows Logons"
+  Text: "See all windows sign-ins"
```
Comment (ms-bnick) [inline Insights/InputEntity_Host/WindowsSigninActivity.yaml:252]: Windows should be capitalized.

File: Insights/InputEntity_Host/EventLogClear.yaml (hunk 2 above)
Comment (shainw) [reply Insights/InputEntity_Host/EventLogClear.yaml:81]: The UX in Event Viewer has it capitalized and also [Win32 API references](https://docs.microsoft.com/en-us/windows/win32/wes/windows-event-log-reference), so we will leave it.

File: Insights/InputEntity_Host/ProcessEntropy.yaml (hunk 5 above)
Comment (shainw) [reply Insights/InputEntity_Host/ProcessEntropy.yaml:190]: Weight is a prominent word here and specifically associated with a column.  We will leave.

File: Insights/InputEntity_Host/WindowsSigninActivity.yaml (hunk 6 above)
Comment (shainw) [reply Insights/InputEntity_Host/WindowsSigninActivity.yaml:252]: Done

File: Insights/InputEntity_Host/GroupAdditions.yaml (hunk 4 above)
Comment (shainw) [reply Insights/InputEntity_Host/GroupAdditions.yaml]: Done

File: Insights/InputEntity_Host/EventLogClear.yaml (hunk 3 above)
Comment (shainw) [reply Insights/InputEntity_Host/EventLogClear.yaml]: Done

File: Insights/InputEntity_Account/WindowsSigninActivity.yaml (hunk 1 above)
Comment (shainw) [reply Insights/InputEntity_Account/WindowsSigninActivity.yaml]: Done

**Pull request merged**